src/io/p5.Table.js

```diff
@@ -1107,7 +1107,7 @@ p5.Table.prototype.getNum = function(row, column) {
  * @return {String}
  *
  * @example
- * <div class="norender">
+ * <div>
```

Comment:
If the `norender` class is not present, the sketch will still create a canvas but it would be empty in this particular case since nothing will be drawn on screen. Can be a bit confusing I think since most people won't have the console opened all the time.